---
title: Distribution and Validation of EPSPS- and PPO-Inhibitor Resistance in *Amaranthus
  palmeri* from Southwestern Nebraska.
author:
- affiliation: University of Wisconsin-Madison
  email: max.oliveira@wisc.com
  name: Maxwel C Oliveira
- affiliation: University of Illinois Urbana-Champaign
  email: dagiac@illinois.edu
  name: Darci A Giacomini
- affiliation: University of Wisconsin-Madison
  email: arsenijevic@wisc.edu
  name: Nikola Arsenijevic
- affiliation: University of Illinois Urbana-Champaign
  email: tranel@illinois.edu
  name: Patrick J Tranel
- affiliation: University of Wisconsin-Madison
  corresponding: rwerle@wisc.com
  email: rwerle@wisc.com
  name: Rodrigo Werle
address:
- address: Department of Agronomy, University of Wisconsin-Madison, Madison, Wisconsin,
    USA
  code: University of Wisconsin-Madison
- address: Department of Crop Sciences, University of Illinois Urbana-Champaign, Urbana,
    Illinois, USA
  code: University of Illinois Urbana-Champaign
output:
  rticles::plos_article: default
  pdf: default
csl: plos.csl
bibliography: bibPalmer.bib
abstract: "Failure to control *Amaranthus palmeri* with 5-enolpyruvylshikimate-3-phosphate
  synthase (EPSPS)- and protoporphyrinogen oxidase (PPO)-inhibitor herbicides was
  reported across southwestern Nebraska in 2017. The objectives of this study were
  to (1) confirm and validate EPSPS (glyphosate)- and PPO (fomesafen and lactofen)-resistance
  in 51 *A. palmeri* populations from southwestern Nebraska using genotypic and whole-plant
  phenotypic assays, and (2) determine which agronomic practices are influencing EPSPS-inhibitor
  resistance in *A. palmeri* populations in that geography. Based on genotypic assay,
  88\\% of 51 populations contained at least one individual with *EPSPS* gene amplification
  (> 2 copies), which confers glyphosate resistance; or a mutation in the *PPX2* gene
  $\\triangle$G210 or R128G/M, endowing PPO-inhibitor resistance in *A. palmeri*.
  High correlation (0.83) between genotypic and phenotypic assays demonstrated that
  *EPSPS* gene amplification is the main glyphosate resistance mechanism in *A. palmeri*
  populations from southwesten Nebraska.  In contrast, there was poor association
  between genotypic and phenotypic responses for PPO resistance, which we attribute
  to a combination of other resistance mechanisms being present, segregation of the
  populations, and using too high of herbicide doses. Thus, genotypic assays could
  expedite the process for confirmation of EPSPS- but not necessarily for PPO-inhibitor
  resistance in *A. palmeri* from southwestern Nebraska. Moreover, *EPSPS* gene amplification,
  county, current and previous crops are the main factors influencing glyphosate resistance
  within that geography. Most of EPSPS-susceptible *A. palmeri* populations were found
  in a few counties in high crop diversity areas. Results presented here confirmed
  the spread of EPSPS- and PPO-inhibitor resistance in *A. palmeri* populations from
  southwestern Nebraska and show that less diverse cropping systems are an important
  driver for EPSPS-inhibitor resistance evolution in *A. palmeri*.  \n"
---
\noindent
**Keywords**: Glyphosate; Fomesafen; Lactofen; Palmer amaranth; Weed resistance.



# Introduction

*Amaranthus palmeri* S. Watson (Palmer amaranth) is a  plant species indigenous to the southwestern United States (US) and northern Mexico  [@sauer_recent_1957]. Despite being an edible plant used to feed animals and Native Americans [@smith_fodder_1900], *A. palmeri* has long been documented as a serious weed problem in US cropping systems [@hamilton_weeds_1958]. *A. palmeri* produces thousands of seeds and grows up to 2 m tall with many lateral branches [@sauer_revision_1955], making it a very competitive species with crops [@morgan_competitive_2001; @massinga_water_2003]. In the 1970s, when cotton picking became mechanized, machinery contributed to the spread of *A. palmeri* seeds across the southern United States [@sauer_dioecious_1972]. At that time, *A. palmeri* was considered the most successful weed of all dioecious *Amaranthus* species as it became widespread in cotton fields [@sauer_dioecious_1972]. Currently, *A. palmeri* is the most economically damaging weed species infesting corn, cotton, and soybean fields in the southern US [@price_glyphosate-resistant_2011; @ward_palmer_2013].

The economic importance of *A. palmeri* is primarily related to its ability to evolve resistance to herbicides. Factors relating to the intrinsic biology of this species have contributed to its fast herbicide resistance evolution [@ward_palmer_2013]. For example, *A. palmeri* reproduces via obligate cross pollination, increasing the chances of hybridization and the exchange of herbicide resistance alleles amongst *Amaranthus* species [@oliveira_interspecific_2018; @gaines_interspecific_2012]. In addition, human-driven selection has strongly contributed to the rise of *A. palmeri* as a problematic weed. In the US cropping systems, no-till is a standard practice amongst growers, and *A. palmeri* thrives in no-till fields due to its small seed size, contributing to the rapid increase of *A. palmeri* individuals in field crops. Also, herbicide resistance in *A. palmeri* drastically increased when weed management strategies shifted from the use of multiple herbicide sites of action (SOA) in a single season to reliance on a single SOA post-emergence herbicide (e.g., glyphosate) [@powles_evolved_2008]. Thus far, *A. palmeri* has evolved resistance to herbicide resistance to multiple SOA [@heap_list_2019-1], which is a major concern as weed management in conventional US cropping systems is largely herbicide dependent.  

The history of herbicide resistance evolution in *A. palmeri* is a result of intense selection pressure from herbicides. In the 1990s, the first documented cases of herbicide resistance were against microtubule- [@gossett_resistance_1992], ALS- [@horak_biotypes_1995], and PSII-inhibitor herbicides [@heap_list_2019-1]. With the introduction of glyphosate-resistant (GR) crops, post-emergence applications of glyphosate became widely used for weed management in soybean, cotton, and corn, resulting in evolution of GR *A. palmeri* [@culpepper_glyphosate-resistant_2006]. GR *A. palmeri* spread across the southern and midwestern US through both independent herbicide selection [@kupper_population_2018] and seed dispersal [@farmer_evaluating_2017; @norsworthy_-field_2014]. The spread of GR *A. palmeri* led to a reevaluation of the use of glyphosate as a sole means of weed control and a push to diversify weed management strategies (e.g., application of other herbicide SOA). The use of 4-hydroxyphenylpyruvate dioxygenase (HPPD)-, protoporphyrinogen oxidase (PPO)- and long chain fatty acids (LCFA)- inhibitor herbicides increased in an attempt to manage GR *A. palmeri*. However, *A. palmeri* populations also evolved resistance to HPPD-, PPO- and LCFA-inhibitor herbicides [@heap_list_2019-1]. New technologies such as auxin-resistant crops may be jeopardized by the newest reports of 2,4-D-resistant *A. palmeri* [@heap_list_2019-1] and the number of *A. palmeri* populations with herbicide resistance to multiple herbicide resistance is also on the rise [@schwartz-lazaro_resistance_2017; @kumar_confirmation_2019]. Therefore, *A. palmeri* herbicide resistance evolution is shrinking the chemical control options for weed management in corn, soybean, and cotton fields within US cropping systems.  

In the US Midwest, corn and soybean growers strongly rely on EPSPS- (e.g., glyphosate) and PPO- (e.g., fomesafen) inhibitor herbicides for weed management. The recent migration of *A. palmeri* into the US Midwest poses a serious threat to the sustainability of crop production in that geography. *A. palmeri* is now overlapping territory with another problematic dioecious *Amaranthus* species, waterhemp (*Amaranthus tuberculatus*). Thus, prevention and/or rapid diagnosis of herbicide resistance in *A. palmeri* has become a priority for agricultural stakeholders. The advances in high-throughput genome sequencing methods are expediting the elucidation and detection of herbicide resistance mechanisms in *A. palmeri* and other weed species. In the case of glyphosate, the most common resistance mechanism in *A. palmeri* is  *EPSPS* gene amplification [@gaines_molecular_2019; @gaines_mechanism_2011], while in PPO-inhibitor resistance, the major resistance mechanism is a *PPO2* glycine 210 deletion ($\triangle$G210) [@salas_resistance_2016; @salas-perez_frequency_2017]. Nonetheless, novel herbicide resistance mechanisms in *A. palmeri* are still being uncovered [@gaines_molecular_2019], as evidenced by the recent documentation of two mutations in the *PPO2* enzyme in the R128 site of *A. palmeri* [@giacomini_two_2017-1], and G399A, an amino acid substitution of glycine to alanine in the catalytic domain of *PPO2* at position 399 [@rangani_novel_2019]. So far, a few reports of resistance mechanisms have been confirmed for *A. palmeri* populations with non-target-site resistance against EPSPS-[@dominguez-valenzuela_first_2017] and one against PPO-inhibitor herbicides [@varanasi_confirmation_2018]. Therefore, using genotypic assays might provide faster detection of known herbicide weed resistance mechanisms in *A. palmeri*, but it fails to address herbicide resistance resulting from novel mechanisms. 

In the fall of 2017, growers in southwestern Nebraska reported failure to control *A. palmeri* with EPSPS- and PPO-inhibitor herbicides (Werle R, personal communication). The objectives of this study were to (1) confirm EPSPS- and PPO-inhibitor resistance in 51 *A. palmeri* populations from southwestern Nebraska via genotypic resistance assays and compare these results via whole-plant phenotypic assay of their progenies, and (2) evaluate agronomic practices that may contribute to EPSPS- and PPO-inhibitor resistance in *A. palmeri* populations.  
 




# Material and Methods


## Plant Material and Growing Conditions

The study was performed with 51 arbitrarily selected *A. palmeri* populations infesting cropping systems across southwestern Nebraska. Each population was collected from a single field. Location, agronomic practices, *A. palmeri* distribution and density of each population were collected (Table 1). In August 2017, green leaf tissues were harvested from five actively growing plants from each of the 51 *A. palmeri* populations, then labeled and stored at -80 C to be used in genotypic assays. Within the 51 *A. palmeri* populations, a second sampling of 19 arbitrarily selected populations was obtained by collecting seeds (progenies) from 10 plants each in September 2017, then cleaned and stored at 5 C until the onset of the greenhouse experiments. Seeds were planted in 900 cm^-3^ plastic trays containing potting-mix (Pro-Mix\textsuperscript \textregistered, Quakertown, PA, USA). Emerged seedlings (1 cm) were transplanted into 164 cm^-3^ cone-tainers. *A. palmeri* plants were supplied with adequate water and kept under greenhouse conditions at 28/20 C day/night temperature with 80% relative humidity. Artificial lighting was provided using metal halide lamps (600 $\mu$mol m^-2^ s^-1^) to ensure 15 h photoperiod.  


## Genotypic Herbicide Resistance Mechanism Assays


Genomic DNA extraction from leaf tissue samples collected from 51 *A. palmeri* populations (five plants per population) were performed using a modified CTAB method [@doyle_rapid_1987]. DNA quality and quantity were checked on a Nanodrop 1000 (Thermo Fisher Scientific, Inc., Waltham, MA, USA) and any samples with low DNA yields or high protein:DNA ratios were discarded and re-extracted. TaqMan qPCR assays were used to check for the presence of known PPO-inhibitor resistance mutations on the *PPO2* enzyme, including the glycine 210 deletion [@wuerffel_distribution_2015] and the R128G/M mutations [@varanasi_statewide_2018]. Samples were also tested for glyphosate resistance via increased numbers of *EPSPS* genomic copies using a SYBR qPCR approach [@chatham_multistate_2015] in which *EPSPS* copy numbers were estimated based on comparison with a single-copy reference gene (*CPS*, carbamoyl phosphate synthetase). 

Herein, for the objective of this study, individuals possessing *EPSPS* copy numbers > 2 are considered EPSPS-resistant and individuals with presence of $\triangle$G210 or R128 het mutations are considered PPO-resistant. Therefore, other target-site and non-target site resistance mechanisms were not tested. 


## Whole-plant Phenotypic Assay

The research was conducted under greenhouse conditions in 2018 and 2019 at the University of Wisconsin-Madison to evaluate the sensitivity of 19 *A. palmeri* populations from southwestern Nebraska to EPSPS- and PPO-inhibitor herbicides. 

The experiments were conducted in a complete randomized design and the experimental unit was a cone-tainer with a single *A. palmeri* seedling. The study was arranged in a factorial design with *A. palmeri* progenies from 19 populations and three herbicides with 20 replications and conducted twice. The arbitrarily selected 19 *A. palmeri* progenies were from Cha 3, Dun 3, Dun 4, Dun 5, Hay 1, Hay 3, Hay 4, Kei 2, Kei 3, Kei 5, Kei 6, Log 1, Log 2, Log 4, Per 2, Per 4, Red 2, Red 4, and Red 5 populations (Table 1). The selected herbicides were glyphosate (Roundup PowerMAX\textsuperscript \textregistered, Bayer Crop Science, Saint Louis, MO, US) applied at 870 g ae ha^-1^ plus 2040 g ha^-1^ ammonium sulfate (DSM Chemicals North America Inc., Augusta, GA); fomesafen (Flexstar\textsuperscript \textregistered, Syngenta Crop Protection, Greensboro, NC, USA) applied at 206 g ai ha^-1^ plus 0.5 L ha^-1^ of non-ionic surfactant (Induce\textsuperscript \textregistered, Helena Agri-Enterprises, Collierville, TN, USA); and lactofen (Cobra\textsuperscript \textregistered, Valent USA LLC Agricultural Products, Walnut Creek, CA, USA) applied at 280 g ai ha^-1^ plus 0.5 L ha^-1^ of non-ionic surfactant.

Herbicide treatments were applied to 8-10 cm tall *A. palmeri* plants with a single-tip chamber sprayer (DeVries Manufacturing Corp., Hollandale, MN, USA). The sprayer had an 8001 E nozzle (Spraying Systems Co., North Avenue, Wheaton, IL, US) calibrated to deliver 140 L ha^-1^ spray volume at 135 kPa at a speed of 2.3 km h^-1^. *A. palmeri* populations were visually assessed 21 days after treatment (DAT) as dead or alive. Plants were considered alive when prominent green tissue was observed in growing plants, while dead plants were completely necrotic.
  

## Statistical Analysis

### Genotypic and Phenotypic validation of EPSPS- and PPO-inhibitor resistance in *A. palmeri*

The number of EPSPS- or PPO-inhibitor-resistant *A. palmeri* individuals in the genotypic assays was converted to a percentage scale:


*Equation 1*: $$G=\frac{S}{T} * 100 $$
\noindent
where *G* is the % EPSPS- or PPO-inhibitor-resistant *A. palmeri* individuals, *S* is the total number of *A. palmeri* individuals positive for herbicide resistance, and *T* is the total number of *A. palmeri* individuals (n=3 to 5) screened for herbicide resistance in genotypic assays. Fomesafen and lactofen are PPO-inhibitor herbicides; thus, *G* is same for both. 

The number of alive *A. palmeri* individuals in the phenotypic assay were converted into a percentage scale:


*Equation 2*: $$P=\frac{X}{T} * 100 $$

Where *P* is the % alive *A. palmeri* individuals after herbicide treatment in phenotypic assay (glyphosate, fomesafen, or lactofen), *X* is the total number of alive *A. palmeri* individuals 21 DAT and *T* is the total number of *A. palmeri* individuals (n=40) treated with each herbicide. The P (%) can be determined only for 19 populations, as only 19 populations were screened to herbicides. Data of two runs are combined.

The correlation between *G* and *P* for each herbicide (glyphosate, fomesafen, and lactofen) and between the two PPO-inhibitor herbicides, fomesafen and lactofen, were performed with Pearson's analysis using the *cor.test* function of R statistical software version 3.6.1 [@r_core_team_r:_2019]. The correlation value varies from -1 and 1, where 1 is the total positive correlation, -1 the total negative correlation, and 0 indicate no linear correlation. Pearson's analysis tests the hypothesis that correlation between two variables is equal to 0 (null hypothesis). If *P*-value > 0.05, the probability is > 5% that a correlation of some magnitude between two variables could occur by chance alone assuming null hypothesis is true; thus, no correaltion between variables. The *G* and *P* correlation was performed with the 19 *A. palmeri* populations treated with herbicide in the phenotypic assay as well as their respective genotypic assays results.  

Although the individuals screened in genotypic (parent) and phenotypic (progeny) assays are not the same; the methodology was chosen to simulated a real farm scenario whereas growers could potentially collect leaf samples and mail them to a laboratory for faster genotypic herbicide resistance confirmation.

### Random Forest 

Random forest is a powerful ensemble machine learning algorithm which generates and combines multiple decision trees to reach a consensus. The random forest procedure is described in detail by Breiman [@breiman_random_2001] and Biau [@biau_random_2016]. In short, the random forest analysis is largely based on two parameters: *ntree*, which is the number of decision trees, and *mtry*, the number of different predictors tested in each tree. For each decision tree, a subsample of observations from the data are selected with replacement to train the trees (bootstrap aggregating). These “in-bag” samples include approximately 66% of the total data and some observations may be repeated in each new training data set since this sampling is done with replacement.  The remaining 33% of the data are designated "Out-of-Bag" or OOB samples and are used in an internal cross-validation technique to estimate the model performance error. To evaluate the importance of an explanatory variable (or predictor), the random forest measures both the decrease in model performance accuracy as calculated by the OOB error and the decrease in the Gini Index value. The Gini Index value (mean decrease in accuracy) is the mean of a total variable decrease of a node impurity, weighted by the samples proportion reaching that node in each individual decision tree. Therefore, variables with a large Gini Index value indicates higher variable importance, and are more important for data classification. Random Forest has been used to described the incidence of crop disease [@langemeier_factors_2016] and glyphosate resistance in *Amaranthus* spp. [@vieira_distribution_2018].

The random forest analysis was performed with the *randomForest* package of R statistical software to describe the influence of *EPSPS* gene amplification (genotypic), PPO-inhibitor resistance (genotypic), location (county), agronomic practices (e.g., tillage, irrigation, current and previous cropping-system), and weed demographics (e.g., density and distribution) on EPSPS-inhibitor resistance in *A. palmeri* in southwestern Nebraska (Table 1). *EPSPS* gene copy number (genotypic) was included as an explanatory variable to test the robustness of random forest as it is highly correlated to glyphosate resistance in *A. palmeri* [@gaines_molecular_2019]. The *A. palmeri* populations with at least one individual with > 2 *EPSPS* copy number was considered resistant (Table 2); therefore, EPSPS-inhibitor resistance was classified as Yes (> 2 *EPSPS* copy number) or No.  For this analysis, the *ntree* parameter was set to 500, whereas *mtry* was set to 2. Parameters numbers selected are standard for the sample size of the present study.


# Results 

## Genotypic and phenotypic validation of EPSPS- and PPO-inhibitor resistance in *A. palmeri*

The *EPSPS* amplification was found in 63% of the 51 *A. palmeri* populations analysed (Table 2). Phenotypic analysis of a subset of these populations confirmed the genotypic analysis data, in that a positive correlation (0.83; *P*-value=0.0000) was observed between *G* and *P* assays (Figure 1 and Table 3). Seven *A. palmeri* populations tested negative (*G*=0) for glyphosate resistance in the genotypic assay but three of these populations showed low (*P*=18%, Hay 1), moderate (*P*=35%, Hay 4) and high (*P*=75%, Red 5) survival after glyphosate treatment in the phenotypic assays (Figure 1). The other four populations (Dun 3, Hay 3, Log 2 and Kei 3) that tested negative (*G*=0%) for EPSPS-inhibitor resistance in the genotypic assay showed less than 15% glyphosate survival (*P*). 

Fomesafen and lactofen resulted in less than 40% survival of *A. palmeri* populations in the phenotypic assays (Figure 2). The correlation between *G* and *P* for PPO-inhibitor resistance in *A. palmeri* populations was inconsistent (Table 3). While a high *G* and *P* correlation (0.52; *P*-value=0.0217) was observed for fomesafen (Figure 2A), no *G* and *P* correlation (-0.05; *P*-value=0.84) was found for lactofen (Figure 2B). *A. palmeri* populations Dun 5, Kei 2, Kei 5, and Log 4 were segregating for PPO resistance in the genotyping assay but individuals in these populations were sensitive to lactofen treatment (*P*=0%, Figure 2B). However, these populations were less sensitive to fomesafen. For example, nearly 30% of individuals from the Log 4 population survived fomesafen treatment (Figure 2A). In contrast, *A. palmeri* populations Cha 3, Kei 6, Per 2, and Red 5 tested negative for PPO-inhibitor resistance (*G*=0%) but over 15% of populations survived both fomesafen and lactofen treatment, suggesting the presence of an alternate resistance mechanism. Also, *A. palmeri* populations Kei 3, Per 4, and Dun 4 showed 38, 25, and 18% survival after fomesafen treatment but less than 15% for lactofen. There was no correlation (0.23; *P*-value=0.34) between fomesafen and lactofen in the phenotypic assay (Table 3). 

According to the genotypic assay, nearly 70% of *A. palmeri* populations from southwestern Nebraska were confirmed resistant to PPO-inhibitor herbicides. In the 34 PPO-inhibitor-resistant *A. palmeri* populations, 32 contain the $\triangle$G210 deletion in the *PPX2* gene, while the R128M/G endowed PPO-inhibitor resistance in two *A. palmeri* populations. Nearly 14% of *A. palmeri* populations had all individuals resistant, 53% were segregating for resistance, and 33% had no mutation detected to PPO-inhibitor herbicides. In addition, based on *EPSPS* gene amplification, our study showed that 10% of *A. palmeri* populations had all individuals resistant to glyphosate, 53% were segregating for resistance, and 37% were susceptible to glyphosate (Table 2). Multiple resistance (EPSPS- and PPO-inhibitor herbicides) was present in 41% of *A. palmeri* populations of southwestern Nebraska (Table 2), while 6, 11, and 13 *A. palmeri* populations were susceptible to both herbicides, resistant to only glyphosate-, and resistant to only PPO-inhibitor herbicides, respectively (Figure 3).


## Random Forest

The final OOB error rate of the random forest analysis was 13.33%, meaning that > 86% of OOB samples were adequately classified by the model. The random forest analysis ranked (high to low) *EPSPS* gene amplification > county > current crop > previous crop > *A. palmeri* density > tillage > irrigation > *A. palmeri* distribution > PPO-inhibitor resistance as the factors influencing the presence of EPSPS-inhibitor resistance in *A. palmeri* of southwestern Nebraska (Figure 4). The selection of *EPSPS* gene amplification as the top predictor shows the robustness of this model, since *EPSPS* gene amplification is known to confer resistance to glyphosate. County was the second most important factor for the presence of EPSPS-inhibitor resistant *A. palmeri*. The presence of EPSPS-inhibitor resistance in *A. palmeri* was found in at least one population of all surveyed counties from southwestern Nebraska (Table 2). The lowest number of EPSPS-inhibitor-resistant populations was found at Hayes (Hay) and Perkins (Per) County with 1 (out 5) and 2 (out 6), respectively. Also, current and previous crops strongly influenced the presence of EPSPS-inhibitor resistance in *A. palmeri* populations. Five *A. palmeri* populations (Dun 4, Dun 5, Fro 1, Kei 2, and Lin 2) had 100% resistant individuals and these populations were all  found in current corn or soybean crops preceded by a corn or sorghum crops (Figure 5A). In contrast, *EPSPS* gene amplification did not occur in 19 *A. palmeri* populations, from which only two populations were found in corn and soybean rotations (Fro 5 and Hit 5; Table 2). The majority of EPSPS-inhibitor-susceptible *A. palmeri* populations were found in rotations of corn, soybean, sorghum, wheat, fallow, and other crops (e.g., alfalfa, dry beans and field peas; Figure 5A). 

Nearly 70% of EPSPS-inhibitor-susceptible *A. palmeri* populations were resistant to PPO-inhibitor herbicides (Table 2). A similar trend of crop diversity was observed for PPO-inhibitor resistance in *A. palmeri*, except a crop rotation of corn-sorghum-soybean was found in locations with only PPO-inhibitor-resistant individuals (Figure 5B). The random forest analysis was performed on EPSPS-inhibitor resistance only due to the robustness of *EPSPS* gene amplification as a positive control to detect resistance in *A. palmeri*, which is different from our results with PPO-inhibitor resistance.

# Discussion

The high correlation between *G* and *P* for EPSPS-inhibitor resistance demonstrates that most GR *A. palmeri* populations from southwestern Nebraska are resistant due to *EPSPS* gene amplification. However, other resistance mechanisms are likely present since a few *A. palmeri* populations showed no *EPSPS* gene amplification despite a majority of the individuals surviving glyphosate (870 g ae ha^-1^) application in the phenotypic assay. *A. palmeri* was the first identified weed to evolve glyphosate resistance via *EPSPS* gene amplification [@gaines_gene_2010], followed by *Kochia scoparia*, *Amaranthus tuberculatus*, *Lolium perenne ssp. multiflorum*, *Bromus diandrus*, *Eleusine indica*, *Chloris truncata*, and *Amaranthus hybridus* [@sammons_glyphosate_2014; @patterson_glyphosate_2018]. The *EPSPS* gene amplification mechanism is widespread in *A. palmeri* [@sammons_glyphosate_2014; @gaines_molecular_2019], albeit other EPSPS-inhibitor resistance mechanisms have arisen, including *Pro106* mutation in the *EPSPS* gene and reduced glyphosate absorption/translocation [@dominguez-valenzuela_first_2017; @sammons_glyphosate_2014]. It remains unknown whether the *A. palmeri* populations (e.g., Red 5) which have low or no *EPSPS* gene amplification harbor additional resistance mechanisms, warranting further investigations.  

*EPSPS* gene amplification is an important evolutionary mechanism enabling weeds and other pests [@remnant_gene_2013; @bass_gene_2011] to evolve resistance to pesticides. Research on *EPSPS* gene amplification molecular basis in weed species is underway but more work is still needed to unveil this complex adaptative trait. The genetics of *EPSPS* gene amplification in weed species follows Mendelian (*K. scoparia*) [@jugulam_tandem_2014] and non-Mendelian (*A. palmeri* [@giacomini_variable_2019] and *B. diandrus* [@malone_epsps_2016]) inheritance patterns. Over 100 *EPSPS* gene copies have been documented in *A. palmeri* but maximum of only 10 have been observed in *K. scoparia* [@wiersma_gene_2015; @kumar_molecular_2015]. The *EPSPS* gene copy variation in *A. palmeri* is a result of the extrachromosomal circular DNA transmitted to the next generation by tethering to mitotic and meiotic chromosomes (eccDNA) [@koo_extrachromosomal_2018], while in *K. scoparia*, *EPSPS* copies are arranged in tandem repeats at a single locus [@patterson_glyphosate_2018]. Segregation for *EPSPS* copy number in *A. palmeri* families (F\scriptsize 1 \normalsize and F\scriptsize 2 \normalsize) is transgressive, with individuals varying in *EPSPS* gene amplification levels even among clonal plants [@giacomini_variable_2019]. Transgressive segregation for *EPSPS* in *A. palmeri* might explain the variable *EPSPS* copy numbers between individuals within populations screened from southwestern Nebraska (Table 2). Gene amplification coupled with the dioecious nature of *A. palmeri* are valuable traits for this weed that help  increase its genetic complexity and allow it to adapt to current US cropping systems.

According to the genotypic assay, most of *A. palmeri* populations from southwestern Nebraska also showed resistance to PPO-inhibitor herbicides. The mechanism of resistance is largely due to the $\triangle$G210 deletion but also due to R128M/G mutations in the *PPX2* gene (Table 2). It has been demonstrated that a mutated PPO enzyme has reduced affinity for several PPO-inhibitor herbicides in *A. palmeri* [@schwartz-lazaro_resistance_2017]. Application of fomesafen (206 g ai ha^-1^) and lactofen (280 g ai ha^-1^) provided high mortality in *A. palmeri* populations from southwestern Nebraska, including *A. palmeri* populations with 100% individuals with $\triangle$G210 deletion. Therefore, it is possible that most of sprayed *A. palmeri* individuals in the phenotypic assay had no mutation for PPO-inhibitor resistance. High *A. palmeri* mortality suggests that populations are segregating for PPO-inhibitor resistance at a low level. Still, there is correlation between *G* and *P* for fomesafen but not lactofen, which was sprayed slightly above the labled rate of 218 g ai ha^-1^. The validation for PPO-inhibitor resistance presented here is limited by the segregating nature of  *A. palmeri* collected from field populations, since individuals in the phenotypic assays were not the same as those screened in the genotypic assays. This could be also explained in several ways, 1) the number of individuals sampled for the *G* assay study may have been too low for the objective of validation; 2) high herbicide rate resulting in high individual mortality; 3) greenhouse conditions were ideal and plants had no stress during application of PPO-inhibitor herbicides, which might be different from plants under field conditions of southwestern Nebraska [@schafer_rhizosphere_2014]; and 4) it is likely that plants were smaller than usual because of small volume of cone-tainers, the size of a plant strongly impacts the level of resistance, with smaller plants showing less resistance than larger plants [@coburn_influence_2017].

Populations of *A. palmeri* have been reported to be resistant to HPPD-[@jhala_confirmation_2014], PSII-[@jhala_confirmation_2014] and EPSPS-inhibitor herbicides [@chahal_glyphosate-resistant_2017] in Nebraska. In 2014, a survey with *Amaranthus* spp. in southwestern Nebraska showed widespread EPSPS-inhibitor resistance for *A. tuberculatus* (81%) but not for *A. palmeri* (6%) [@vieira_distribution_2018]. Our survey showed a high number of EPSPS-inhibitor-resistant/segregating *A. palmeri* populations in southwestern Nebraska (>60%). The rapid evolution of EPSPS-inhibitor-resistant *A. palmeri* in southwestern Nebraska raised questions about whether populations were introduced via seed/gene flow or arose independently. Although we did not test the hypothesis, the random forest analysis did shed some light on EPSPS-inhibitor resistance evolution in *A. palmeri*. The random forest analysis results suggested that *EPSPS* gene amplification, county, and current and previous crops were the top factors influencing GR *A. palmeri* in southwestern Nebraska. The *EPSPS* gene amplification as the primary factor for glyphosate resistance in *A. palmeri* showed the robustness of random forest analysis, while county influence on *EPSPS*-inhibitor resistance in *A. palmeri* highlighted how practices for delaying EPSPS-inhibitor resistance evolution should start at the landscape level. It is likely that growers with common goals may implement best weed management tactics to delay EPSPS-inhibitor resistance evolution and prevent seed/gene flow of herbicide resistant weeds. Nonetheless, it is striking that PPO-inhibitor resistance was common in counties with low/no EPSPS-inhibitor resistance. For instance, most EPSPS-inhibitor-susceptible *A. palmeri* were found in Hayes County in areas of sorghum or corn in rotation with wheat (Table 1), which are less glyphosate-dependent crops. Although best weed management practices at the landscape level is encouraged, our results suggest that growers were not necessarily implementing good management practices but rather, using less EPSPS-inhibitor herbicide.

The high presence of *A. palmeri* populations with 100% resistance to EPSPS-inhibitors in less diverse cropping systems (e.g., corn-soybean rotation) suggests the influence of repeated glyphosate application on *A. palmeri* evolution to EPSPS-inhibitor resistance. The occurrence of EPSPS-inhibitor-resistant *A. palmeri* populations is reduced in rotations with more diversified crops (Figure 5A), most likely due to rotations with glyphosate sensitive crops (no/less glyphosate use). Crop diversity exerts a different selection pressure on weed communities, including canopy closure timing, seeding, and harvest date, which aid on reducing the dominance of single weed species [@andrade_weed_2017]. Nonetheless, it is likely that Nebraska growers still rely on other herbicide sites of action for weed management, as most EPSPS-inhibitor-susceptible *A. palmeri* populations were resistant to PPO-inhibitor herbicides (e.g., fomesafen). Similar to EPSPS-inhibitor resistance, the PPO-inhibitor-resistant *A. palmeri* was also found more often in less diverse crop rotations (Figure 5B). Despite not having long-term herbicide application records for the surveyed area, it is possible that overreliance on a single or few herbicide SOA in areas with low crop diversity contributed to resistance. In addition, it has been shown that herbicide mixture (multiple SOA in one application) is more effective for delaying herbicide weed resistance than herbicide rotation (multiple applications, each with a single SOA) [@evans_managing_2016; @beckie_selecting_2009]. Thus, it is likely that without herbicide mixture, increased crop diversity is not enough to reduce the risk of herbicide resistance evolution in weed species.

An emerging concern in weed science is the ability of some species to stack genes for multiple herbicide resistance in a single population. Five- and six-way herbicide resistance has been reported in an *A. tuberculatus* population from Illinois [@strom_characterization_2019] and Missouri [@shergill_investigations_2018], respectively. The 19 *A. palmeri* populations used in the whole-plant phenotypic also tested positive (>80% of individuals) to an ALS-inhibitor herbicide (imazethapyr [70 g ai ha^-1^]; data not shown). Moreover, nearly half of the *A. palmeri* populations in this study possessed multiple herbicide resistance (EPSPS and PPO; Figure 3). Thus, it is likely that two- and three-way resistance exists in most *A. palmeri* populations from southwestern Nebraska. The random forest analysis suggested no/low influence of PPO-inhibitor resistance on EPSPS-inhibitor resistance, indicating *EPSPS* gene amplification and PPO-inhibitor resistance mutations ($\triangle$G210 or R128M/G) are not genetically linked. Cosegregation of distinct SOA resistance mechanisms is unlikely but not impossible as it has been shown that ALS- and PPO-inhibitor resistance are genetically linked in a *A. tuberculatus* population [@tranel_target-site_2017]. 

Herein we demonstrate the widespread occurrence of EPSPS- and PPO-inhibitor-resistant *A. palmeri* in southwestern Nebraska. Glyphosate resistance via *EPSPS* gene amplification was present in the majority of *A. palmeri* populations from Nebraska, supporting the use of genotypic assays for faster detection of *EPSPS*-inhibitor resistance in this region. Novel or non-target glyphosate resistance mechanisms may also be present in few *A. palmeri* populations, but at a relatively low frequency. PPO-inhibitor resistance was also present in these populations, but was less correlated with known PPO-inhibitor resistance mutations, warranting the use of both genotypic and whole-plant bioassays for testing PPO-inhibitor resistance. The EPSPS- or PPO-inhibitor-resistant *A. palmeri* populations were found in areas of low crop diversity, suggesting resistance evolution was mostly due to high selection pressures of EPSPS- and PPO-inhibitor herbicides. Great progress has been made towards understanding the *A. palmeri* molecular basis of resistance, but the continuous spread of herbicide resistance in *A. palmeri* populations to new geographies is evident. Thus, there is a need to change the current methods for crop and weed management, including dependence on corn/soybean rotation, and bring non-herbicide weed management innovations to US cropping systems.

# Acknowledgments

This research received no specific grant from any funding agency. We also appreciate the help of Victor Ribeiro's assistance with the greenhouse projects. No conflicts of interest have been declared.

# References {#references .unnumbered}


\begin{table}[!ht]  \small
\begin{adjustwidth}{-2.5in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf Demographic list of \textit{Amaranthus palmeri} populations with respective Nebraska County location and agronomic practices}}
\begin{tabular}{|l+c|c|c|c|c|c|c|c|}
\hline
{\bf Population}  & {\bf County} & {\bf Current crop} & {\bf Previous crop$^a$} & {\bf Tillage} & {\bf Irrigation} & {\bf Weed distribution} & {\bf Weed density}\\ \thickhline
Cha 1 & Chase & sorghum  & corn & tilled & rainfed & spread & Low\\ \hline
Cha 2 & Chase   & corn & wheat  & strip-till & center pivot &   spread & high\\ \hline
Cha 3   & Chase & corn &    fallow/cornstalks   & no-till   & centerpivot   & spread &  high\\ \hline
Cha 4   & Chase & soybeans &    fallow/cornstalks   & no-till   & centerpivot   & spread &  low\\ \hline
Cha 5   & Chase & corn &    corn    & strip-till    & rainfed & spread &    low\\ \hline
Dun 1   & Dundy & wheatstubble &    other$^b$ & tilled &    rainfed &   spread &    intermediate\\ \hline
Dun 2   & Dundy & corn &    sorghum &   no-till &   rainfed &   spread &    intermediate\\ \hline
Dun 3   & Dundy &   other   &    & tilled & centerpivot &   spread &    intermediate\\ \hline
Dun 4   & Dundy & corn &    corn &  no-till &   centerpivot &   edges & high\\ \hline
Dun 5   & Dundy &   soybeans &  corn &  tilled &    centerpivot &   spread &    low\\ \hline
Fro 1   & Frontier &    corn &  sorghum & no-till   & rainfed & edges & high\\ \hline
Fro 2   & Frontier &    soybeans &  corn &  tilled &    rainfed &   edges   & low\\ \hline
Fro 3   & Frontier &    soybeans &  wheatstubble &  tilled &    centerpivot & spread &  high\\ \hline
Fro 4   & Frontier &    sorghum  & fallow/cornstalks &  tilled &    rainfed  & edges &  intermediate\\ \hline
Fro 5   & Frontier &    soybeans &  corn &  tilled &    centerpivot & edges & high\\ \hline
Hay 1   & Hayes & sorghum & fallow/cornstalks    & tilled & centerpivot  & spread & intermediate\\ \hline
Hay 2   & Hayes & corn &    wheatstubble &  no-till &   rainfed  & spread & intermediate\\ \hline
Hay 3   & Hayes & sorghum    & wheatstubble & tilled &  centerpivot &   spread &    high\\ \hline
Hay 4   & Hayes & corn &    wheatstubble &  no-till &   rainfed  & edges &  intermediate\\ \hline
Hay 5   & Hayes & sorghum & wheatstubble &  no-till &   rainfed &   spread &    high\\ \hline
Hit 1   & Hitchcock & corn &    fallow/cornstalks & tilled &    centerpivot &   edges & low\\ \hline
Hit 2   & Hitchcock & soybeans &    corn &  no-till &   rainfed &   spread &    low\\ \hline
Hit 3   & Hitchcock & corn &    corn &  no-till & rainfed   & edges & high\\ \hline
Hit 4 & Hitchcock   & sorghum & wheatstubble &  no-till &   rainfed &   edges & high\\ \hline
Hit 5   & Hitchcock & soybeans &    corn    & no-till & centerpivot &   edges   & high\\ \hline
Kei 1   & Keith & other &   fallow/cornstalks & tilled &    centerpivot &   spread &    high\\ \hline
Kei 2   & Keith & corn &    fallow/cornstalks & no-till &   centerpivot  & spread & intermediate\\ \hline
Kei 3   & Keith & soybeans &    &   tilled &    furrow &    spread &    high\\ \hline
Kei 4   & Keith & soybeans &      & no-till &   centerpivot  & spread & low\\ \hline
Kei 5   & Keith & other &   corn &  tilled  & centerpivot   & spread &  low\\ \hline
Kei 6   & Keith & soybeans &    &   no-till &   centerpivot &   spread &    high\\ \hline
Lin 1   & Lincoln & corn &  other & no-till & centerpivot & spread &    high\\ \hline
Lin 2   & Lincoln     & soybeans &  corn &  tilled &    centerpivot &   spread  & low\\ \hline
Lin 3   & Lincoln   & soybeans &    &   tilled &    centerpivot &   spread &    low\\ \hline
Lin 4   & Lincoln   & corn &    &   tilled &    furrow &    spread &    high\\ \hline
Lin 5   & Lincoln   & corn &    wheatstubble     & no-till &    rainfed &   spread &    high\\ \hline
Log 1   & Logan  & soybeans &   fallow/cornstalks & tilled &    centerpivot &   edges & intermediate\\ \hline
Log 2   & Logan  & other &  fallow/cornstalks    & no-till &    rainfed  & spread & intermediate\\ \hline
Log 3   & Logan &   soybeans &  corn &  tilled &    centerpivot &   edges & high\\ \hline
Log 4   & Logan &   soybeans &  corn &  tilled &    rainfed  & spread & low\\ \hline
Per 1   & Perkins & other & sorghum &   no-till &   rainfed & spread &  low\\ \hline
Per 2   & Perkins & soybeans &  corn &  strip-till &    centerpivot &   spread &    intermediate\\ \hline
Per 3   & Perkins & fallow/cornstalks & corn &  tilled &    rainfed &   spread &    high\\ \hline
Per 4   & Perkins & soybeans &  corn &  no-till &   centerpivot &   spread &    high\\ \hline
Per 5   & Perkins & other    & fallow/cornstalks &  no-till &   centerpivot &   spread &    intermediate\\ \hline
Per 6   & Perkins   & other & fallow/cornstalks &   no-till  & centerpivot &    spread &    high\\ \hline
Red 1   & Red Willow &  soybeans &  corn &  no-till & centerpivot & edges & high\\ \hline
Red 2   & Red Willow &  corn &  corn &  tilled &    centerpivot &   edges & high \\ \hline
Red 3   & Red Willow &  wheatstubble &  wheat    & no-till &    rainfed &   spread &    intermediate\\ \hline
Red 4   & Red Willow &  corn &  corn &  no-till &   rainfed & spread &  low\\ \hline
Red 5   & Red Willow &  fallow/cornstalks & corn    & no-till & rainfed &   spread &    high\\ \hline
\end{tabular}
\begin{flushleft}
\end{flushleft}
\label{table1}
\footnotesize{$^a$ Blank cells means unidentified crop, $^b$alfalfa, dry beans and field peas}\\
\end{adjustwidth}
\end{table}

\begin{table}[!ht]  \small
\begin{adjustwidth}{-2.5in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf List of \textit{Amaranthus palmeri} populations with EPSPS gene amplification and PPO resistance according to genotypic resistance assays.}}
\begin{tabular}{|l+c|c|c|p{1.8cm}|c|p{1.8cm}|c|}
\thickhline
\multicolumn{1}{|c|}{\bf population} & \multicolumn{4}{c}{\bf EPSPS gene amplification}  & \multicolumn{2}{|c|}{\bf PPO resistance$^a$}  & \multicolumn{1}{c|}{\bf \# Plants$^b$}\\ 
\cline{2-5}\cline{6-7}
{\bf }  & {\bf Mean} & {\bf Max.} &  {\bf Min.} & {\bf \% EPSPS resistant plants} & {\bf Mutation} & {\bf \% PPO resistant plants} & {\bf }\\ \thickhline
Cha 1 & 7 & 23 & 1 & 25 &  & 0 & 4\\ \hline
Cha 2 & 1 & 3 & 1   & 20 &  & 0 & 5\\ \hline
Cha 3   & 9 & 15 & 1 & 80   &  & 0 & 5\\ \hline
Cha 4   & 10 & 26 & 1   & 40 & R128 het & 20 & 5\\ \hline
Cha 5   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Dun 1   & 5 & 18 & 1    & 60 & $\triangle$G210 & 20 & 5\\ \hline
Dun 2   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Dun 3   & 1 & 1 & 1 & 0 & $\triangle$G210 & 67 & 3\\ \hline
Dun 4   & 6 & 10 & 4 & 100 &  & 0 & 5\\ \hline
Dun 5   & 24 & 51 & 4 & 100 &   $\triangle$G210 & 20 & 5\\ \hline
Fro 1   & 6 &   10 & 3 & 100 & $\triangle$G210 & 33 & 3\\ \hline
Fro 2   & 3 & 6 & 1 & 33 & $\triangle$G210 & 100 & 3\\ \hline
Fro 3   & 5 & 11 & 1 & 33 & $\triangle$G210 & 67 & 3 \\ \hline
Fro 4   & 1 & 1 & 1 & 0 &   $\triangle$G210 & 100 & 3\\ \hline
Fro 5   & 1 & 2 & 1 & 0 &    & 0 & 5\\ \hline
Hay 1   & 1 & 1 & 1 & 0 & $\triangle$G210 & 100 & 3\\ \hline
Hay 2   & 2 & 3 & 1 & 33 & $\triangle$G210  & 33 & 3\\ \hline
Hay 3   & 2 & 2 & 1 & 0 & $\triangle$G210 & 100 & 3\\ \hline
Hay 4   & 1 & 1 & 1 & 0 &   $\triangle$G210 & 67 & 3\\ \hline
Hay 5   & 1 & 1 & 1 & 0 &  & 0 & 5\\ \hline
Hit 1   & 5 & 20 & 1 & 20 &  & 0 & 5\\ \hline
Hit 2   & 21    & 57 & 3 & 67 &  & 0 & 3\\ \hline
Hit 3   & 3 & 6 & 1 & 33 & $\triangle$G210 & 33 & 3\\ \hline
Hit 4 & 2   & 3 & 1 & 25 &  & 0 & 4\\ \hline
Hit 5   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Kei 1   & 13    & 38 & 1 & 33 & $\triangle$G210 & 33 & 3\\ \hline
Kei 2   & 12    & 19 & 7 & 100 & $\triangle$G210 & 33 & 3\\ \hline
Kei 3   & 1 & 1 & 1 & 0 &  & 0 & 5\\ \hline
Kei 4   & 8 & 18 & 1 & 60 &  & 0 &  5\\ \hline
Kei 5   & 5 & 8 & 1 & 67 & $\triangle$G210 & 67 & 3\\ \hline
Kei 6   & 17    & 40 & 2 & 80 &  & 0 & 5\\ \hline
Lin 1   & 1 & 2 & 1 & 0 & $\triangle$G210 & 100 & 3\\ \hline
Lin 2   & 5 & 6 & 3 & 100 & $\triangle$G210 & 67 & 3\\ \hline
Lin 3   & 4 & 6 & 1 & 67 & $\triangle$G210 & 100 & 3\\ \hline
Lin 4   & 3 & 6 & 1 & 33 & $\triangle$G210 & 100 & 3\\ \hline
Lin 5   & 1 & 1 & 1 & 0 &  & 0 & 5\\ \hline
Log 1   & 34    & 57 & 1 & 67 & $\triangle$G210 & 33 &  3\\ \hline
Log 2 & 1   & 1 & 1 & 0 &  & 0 & 3\\ \hline
Log 3   & 4 & 7 & 1 & 67 & $\triangle$G210 & 33 & 3\\ \hline
Log 4   & 3 & 6 & 1 & 67 & $\triangle$G210 & 67 & 3\\ \hline
Per 1   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Per 2   & 32    & 59 & 1 & 80 &  & 0 & 5\\ \hline
Per 3   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Per 4   & 10    & 22 & 1 & 67 &  & 0 &  3\\ \hline
Per 5   & 1 & 2 & 1 & 0 & $\triangle$G210 & 33 &  3\\ \hline
Per 6   & 1 & 2 & 1 & 0 & $\triangle$G210 & 33 &  3\\ \hline
Red 1   & 2 & 3 & 1 & 33 & $\triangle$G210 & 33 &  3\\ \hline
Red 2   & 2 & 3 & 1 & 33 & $\triangle$G210 & 67 &  3\\ \hline
Red 3   & 2 & 6 & 1 & 20 & R128 het & 20 &  5\\ \hline
Red 4   & 2 & 5 & 1 & 33 & $\triangle$G210 & 67 &  3\\ \hline
Red 5   & 1 & 2 & 1 & 0 &  & 0 &  5\\ \hline
\end{tabular}
\begin{flushleft}
\end{flushleft}
\label{table1}
\footnotesize{$^a$ blank cells means no PPO resistance found, $^b$ Number of plants screened in the genotypic herbicide resistance assay}\\
\end{adjustwidth}
\end{table}

\begin{table}[!ht]
\begin{adjustwidth}{-2.5in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf Correlation estimates between genotypic (\textit{G}) and whole-plant phenotypic (\textit{P}) assays of glyphosate, fomesafen, lactofen, and between \textit{P} fomesafen and \textit{P} lactofen (PPO inhibitors).}}
\begin{tabular}{|l+c|c|c|c|c|c|}
\hline
{\bf Herbicide}  & {\bf Correlation variables} & {\bf Estimate} & {\bf lower CI$^a$} & {\bf upper CI} & {\bf t} & {\bf \textit{P}-value} \\ \thickhline
glyphosate & \textit{G} and \textit{P} & 0.83 & 0.60  & 0.93 & 6.15 & 0.0000\\ \hline
fomesafen &  \textit{G} and \textit{P}    & 0.52    & 0.09 & 0.79   & 2.53 & 0.0217\\ \hline
lactofen    &  \textit{G} and \textit{P}    & -0.05 & -0.49 &   0.41    & -0.20 & 0.8412\\ \hline
PPO inhibitors  & \textit{P}-fomesafen and \textit{P}-lactofen   & 0.23 & -0.25 &   0.62 & 0.98 & 0.3428\\ \hline
\end{tabular}
\begin{flushleft}
\end{flushleft}
\label{table1}
\footnotesize{$^a$ Confidence interval}\\
\end{adjustwidth}
\end{table}



